node.gyp

```diff
@@ -207,7 +214,9 @@
         'src/udp_wrap.cc',
         'src/util.cc',
         'src/uv.cc',
+        'src/loader/module_wrap.cc',
         # headers to make for a more pleasant IDE experience
+        'src/loader/module_wrap.h',
```

Comment:
@addaleax was for my own organization but we can keep with all files in a single dir if you wish.